packages/@okta/vuepress-site/guides/sign-into-spa/section09/index.md

```diff
@@ -1,25 +1,21 @@
 ---
 title: Require Authentication
 ---
-## Require Authentication
 
-An important decision regarding your authentication requirements is which parts of your application require the user to be authenticated. You can require authentication across the entire app or just require it for particular routes or controllers. There is also the issue of what sort of app behavior you want for unauthenticated users (users who aren't signed in).
+In most applications, you'll want to prevent the user from accessing certain routes or sections unless they are signed in. You can require authentication across the entire app, or just require it for particular routes or controllers. Any route that doesn't specifically require authentication will be accessible without signing in (also called anonymous access).
```

Comment:
In most applications, you want to prevent the user from accessing certain routes or sections unless they are signed in. You can require authentication across the entire app or just require it for particular routes or controllers. Any route that doesn't specifically require authentication is accessible without signing in (also called anonymous access).